### PR TITLE
Integrate qgis plugin

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -185,6 +186,15 @@ docker-build-build: $(shell docker-required --path . --replace-pattern='^test(.*
...
 	docker build --build-arg=VERSION=$(VERSION) --tag=$(DOCKER_BASE)-build:$(MAJOR_VERSION) .
 
+docker/qgisserver/commons: commons
+	rm --recursive --force $@
+	cp --recursive $< $@
+	touch $@
+
+.PHONY: docker-build-qgisserver
+docker-build-qgisserver: $(shell docker-required --path docker/qgisserver) docker/qgisserver/commons
+	docker build --tag=$(DOCKER_TEST_BASE)-qgisserver:latest docker/qgisserver
```
Comment (pvalsecc) [inline Makefile]: Wrong tag...

Comment (sbrunner) [reply Makefile]: Fixed, thanks :-)

File: Makefile [hunk 2]
```diff
@@ -185,6 +186,16 @@ docker-build-build: $(shell docker-required --path . --replace-pattern='^test(.*
 		$(APPS_FILES_ALT)
 	docker build --build-arg=VERSION=$(VERSION) --tag=$(DOCKER_BASE)-build:$(MAJOR_VERSION) .
 
+docker/qgisserver/commons: commons
+	rm --recursive --force $@
+	cp --recursive $< $@
+	touch $@
```
Comment (arnaud-morvan) [inline Makefile]: I find simpler to use repository root as docker context

File: docker/qgisserver/Dockerfile [hunk 3]
```diff
@@ -1,4 +1,7 @@
-FROM camptocamp/qgis-server:3
+FROM camptocamp/qgis-server:latest
 LABEL maintainer Camptocamp "info@camptocamp.com"
 
-COPY * /var/www/plugins/geomapfish_qgisserver/
+COPY __init__.py accesscontrol.py metadata.txt /var/www/plugins/geomapfish_qgisserver/
```
Comment (arnaud-morvan) [inline docker/qgisserver/Dockerfile]: I would create a folder for the plugin

File: Makefile (hunk 2 above)
Comment (sbrunner) [reply Makefile]: No, we relay don't wants to mix everything ...

File: docker/qgisserver/Dockerfile (hunk 3 above)
Comment (sbrunner) [reply docker/qgisserver/Dockerfile]: Done :-)

**Pull request merged**